rerun_py/src/utils.rs

```diff
@@ -23,3 +23,26 @@ where
     let runtime: &Runtime = get_tokio_runtime();
     py.allow_threads(|| runtime.block_on(f))
 }
+
+pub fn py_rerun_warn(msg: &std::ffi::CStr) -> PyResult<()> {
+    Python::with_gil(|py| {
+        let warning_type = PyModule::import(py, "rerun")?
+            .getattr("error_utils")?
+            .getattr("RerunWarning")?;
+        PyErr::warn(py, &warning_type, msg, 0)?;
+        Ok(())
+    })
+}
+
+pub fn py_log_warn(msg: &str) -> PyResult<()> {
```

Comment:
the naming distinction of those two functions is totally intransparent to me. Let's call them both `py_rerun_warn` and rename the cstr one to `py_rerun_warn_cstr`

Reply:
I will add a line documenting their purposes to avoid confusion

Reply:
doesn't that mean the user sees the warning twice? sounds pretty bad

Reply:
Only once, unless the warnings are catched and displayed on the python side.
I believe you might want to inititate a discussion on the practices of this area, the code doesn't have a conistancy of dealing with it and there're around 5 warnings on the rust side only one of them can be catched on the python side. All other warnings are on the python side (I didn't check if the python warnings follow the same pattern of the rust warnings or not).

Reply:
tbh no idea about the rest, and yes that's out of scope. But practically speaking if I'm in the SDK and make the dupl-init mistake do I see the warning pop up twice on the console? If no then that's fine, but I'm very much confused to why not